faiss/IndexFlatCodes.h

```diff
@@ -34,7 +34,13 @@ struct IndexFlatCodes : Index {
 
     void reset() override;
 
-    /// reconstruction using the codec interface
+    /** Reconstruct vectors i0 to i0 + ni - 1
+     *
```

Comment:
please move this doc to the parent as it applies to all children

Reply:
Got it, it confused me a bit because Index.h already has this doc, so I think I should just remove the inline doc.